static/app/views/performance/data.tsx

```diff
@@ -345,10 +347,10 @@ export const PERFORMANCE_TERMS: Record<PERFORMANCE_TERM, TermFormatter> = {
     t(
       'Cumulative layout shift (CLS) is a web vital measuring unexpected visual shifting a user experiences.'
     ),
-  userMisery: organization =>
+  userMisery: () =>
     t(
       "User Misery is a score that represents the number of unique users who have experienced load times 4x your organization's apdex threshold of %sms.",
-      organization.apdexThreshold
+      DEFAULT_PROJECT_THRESHOLD_METRIC
```

Comment:
Won't this mean the tooltip will be wrong when a threshold is set since it's always showing the default threshold? Should this copy be updated to reflect that the threshold is customizable?

Reply:
I have two other PRs to clean this up!
https://github.com/getsentry/sentry/pull/32125
https://github.com/getsentry/sentry/pull/32126